api/src/main/java/com/epam/pipeline/manager/cloud/gcp/GCPClient.java

```diff
@@ -74,6 +79,23 @@ public IAMCredentials buildIAMCredentialsClient(final GCPRegion region) throws I
                 .build();
     }
 
+    public Storage buildStorageClient(final GCPRegion region) {
+        if (StringUtils.isBlank(region.getAuthFile())) {
+            return StorageOptions.getDefaultInstance().getService();
+        }
+        try (InputStream stream = new FileInputStream(region.getAuthFile())) {
+            final GoogleCredentials sourceCredentials = ServiceAccountCredentials
```

Comment:
Can't we just use `buildCredentials` method here? I think it will work fine for both: application default credentials and auth_file?